Index: src/main/java/org/courseregistration/dao/GenericDAO.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- src/main/java/org/courseregistration/dao/GenericDAO.java	(revision )
+++ src/main/java/org/courseregistration/dao/GenericDAO.java	(revision )
@@ -108,7 +108,7 @@
             int counter = 1;
             for (T entity : entities) {
                 entityManager.persist(entity);
-                if ((counter % 1000) == 0) {
+                if ((counter % 10000) == 0) {
                     entityManager.flush();
                     entityManager.clear();
                 }
